source/thanks.html.erb

```diff
@@ -0,0 +1,17 @@
+---
+title: Thanks!
+---
+
+<h1>Thanks!</h1>
+<p>We know that everyone works hard for their money, and we deeply appreciate your decision to help.</p>
+
+<p>Your donation supports the continued work of the Grafeas Group in our efforts for worldwide internet accessibility. Right now, donations are aimed at expanding our partner relationships and increasing our online presence.</p>
+
+<p>Your donation is tax-deductable; once you donate, your receipt will be emailed to you within 24 hours from our partner, Stripe.</p>
+
+<p>Once again, you have our thanks -- thanks for helping us increase accessibility on the web and thanks for being you.</p>
+
+<p>Cheers,</p>
+
+<p>~ Joe Kaufeld<br>
+President, Grafeas Group Ltd.</p>
```

Comment:
Newlines, Joe! Don't forget the newlines at the ends of the files! :smile:

Reply:
Dammit, betrayed again by my editor! :shakes_fist_at_sky: